packages/help-center/src/hooks/use-chat-status.ts

```diff
@@ -27,7 +27,7 @@ export default function useChatStatus(
 	const { status: zendeskStatus } = useZendeskConfig( isEligibleForChat );
 
 	return {
-		canConnectToZendesk: zendeskStatus !== 'error',
+		canConnectToZendesk: zendeskStatus !== 'error' && zendeskStatus !== 'loading',
```

Comment:
It's important to only return `true` after loaded state. Before this change it was returning true while loading, and [that would load chat](https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/hooks/use-zendesk-messaging.ts#L53) even if in an error state was present after loading it.